content/concept-history-sharding.md

```diff
@@ -0,0 +1,56 @@
+# History Sharding
+
+As servers operate, they naturally produce a database containing the nodes from the ledgers they witnessed or acquired during network runtime. This ledger data is stored by the node store, but the online delete logic rotates these databases when their size exceeds configured space limitations.
```

Comment:
> when their size exceeds configured space limitations

Would be cool, but at the moment it is only possible to configure a number of stored ledgers there, right?

Reply:
@MarkusTeufelberger Yes, you are correct.

Reply:
Ah, interesting. OK, changing to:

> Each `rippled` server stores ledger data in its node store, but the online delete logic rotates these databases when the number of stored ledgers exceeds configured space limitations.